src/main/java/domain/Buyer.java

```diff
@@ -0,0 +1,49 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+public class Buyer {
+
+  private Money money;
+  private List<Lotto> lottos = new ArrayList<>();
+
+  public Buyer(Money money) {
+    this.money = money;
+    createLottos();
+  }
+
+  public int getLottoSize() {
+    return lottos.size();
+  }
+
+  public void createLottos() {
+    int totalLotto = money.calculateTotalLotto();
+    for (int i = 0; i < totalLotto; i++) {
+      lottos.add(new Lotto());
+    }
+  }
+
+  public String createResult() {
+    String result = "";
+    for (Lotto lotto : lottos) {
+      result += lotto.toString() + "\n";
+    }
+    return result;
+  }
+
+  public HashMap<LottoMatch, Integer> countLottos(WinningLotto winningLotto) {
```

Comment:
리턴 타입이 Map의 하위 구현체 중 하나인 HashMap을 직접 리턴하고 계시군요? 
가능하다면 유연성과 확장성을 위해서 상위의 인터페이스인 Map을 반환해주는 것을 지향보시면 좋을 것 같아요. 

```suggestion
  public Map<LottoMatch, Integer> countLottos(WinningLotto winningLotto) {
```